product_customer_code/models/product_customer_code.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+# Copyright 2017 Vauxoo (https://www.vauxoo.com) info@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from openerp import fields, models
+
+
+class ProductCustomerCode(models.Model):
+
+    _name = "product.customer.code"
+    _description = "Add manies Code of Customer's"
```

Comment:
Change in [847c3e6](https://github.com/Vauxoo/addons-vauxoo/pull/1206/commits/847c3e6d317d3389ac56c1db63e4cfc53f7a8742)